lib/venue.rb

```diff
@@ -1,2 +1,30 @@
 class Venue
+  attr_reader :name,
+              :capacity,
+              :patrons
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def add_patron(patron)
+    @patrons << patron
+  end
+
+  def yell_at_patrons
+    @patrons.map do |patron|
+      patron.upcase
+    end
+  end
+
+  def over_capacity?
+    @patrons.length > 3
```

Comment:
So you are hard coding the value of 3. When would we actually be over capacity? What variable holds that value? Hint: what is `@capacity`?